proposals/0409-exportable-named-default.rst

```diff
@@ -267,45 +271,78 @@ GHC already supports two extensions that modify the defaulting mechanism:
 ExtendedDefaultRules
 ~~~~~~~~~~~~~~~~~~~~
 
-The former is fully devoted to defaulting. Its effect is to extend the defaulting rules so that they apply not only to
-the class ``Num`` as specified by the language standard, but also to any class in the following list: ``Show``,
-``Eq``, ``Ord``, ``Foldable``, ``Traversable``, or any numeric class. This list is hard-coded and not
-user-extensible. Furthermore, the extension adds ``()`` and ``[]`` to the list of default types to try. If the present
-proposal is accepted, ``ExtendedDefaultRules`` could be reformulated as a set of actual ``default`` declarations
-brought into the scope::
+The ``ExtendedDefaultRules`` extension is fully devoted to defaulting. It changes
+the following:
+
+1. It relaxes the rules for defaulting. Instead of the Haskell98/Haskell2010 rules,
```

Comment:
With the above specification, this section can perhaps just offer some examples?